### PR TITLE
Animated splash screen

### PR DESCRIPTION
**Pull request by aliarbaaz**
# Pull Request

## Description

Added the animated splash screen
Added launcher icon
Applied font

## Checklist

- [x] I have added a screenshot or screen recording showing the functionality of the app.
- [x] I have tested the project and confirmed it works as expected.

## Screenshots

<a href="https://github.com/user-attachments/assets/0dba2b69-480e-45d7-8412-25cb5c16f0be"> Launcher Icon </a>

<a href="https://github.com/user-attachments/assets/7374b491-09c1-4e1d-9227-818a0bd95e1f"> Animated splash screen </a>

### CODE REVIEW

File: meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/custom_views/MealFilledButton.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ) {
+        Text(
+            text = text,
+            color = White,
+        )
+    }
+}
+
+
+
```
Comment (Siduchels) [inline meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/custom_views/MealFilledButton.kt]: Remove extra space...

File: meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/custom_views/MealIconButton.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = text,
+                color = White
+            )
+        }
+    }
+}
+
+
```
Comment (Siduchels) [inline meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/custom_views/MealIconButton.kt]: Remove extra space.

**Review by Siduchels — APPROVED** [review]
Looks good

**Review by aliarbaaz — COMMENTED** [review]
removed extra spaces

**Review by ZSank — APPROVED** [review]
Looks good

**Pull request merged**